setup.py

```diff
@@ -39,7 +39,7 @@ def get_version() -> str:
         "marshmallow>=3.18.0,<3.19.0",
         "marshmallow-dataclass>=8.5.8,<8.6.0",
         "oauthlib>=3.2.1,<3.3.0",
-        "pygitguardian>=1.5.0,<1.6.0",
+        "pygitguardian @ git+https://github.com/GitGuardian/py-gitguardian.git@alina/include-incident-url",  # TODO fix
```

Comment:
The benchmark broke because it installs dependencies using `pipenv install --system` (see .github/workflows/perfbench.yml, in the "Install dependencies" step). I think you can fix this by replacing `--system` with `--skip-lock`.